### PR TITLE
Braitenburg + stopmotors script

### PR DESCRIPTION
**Pull request by dariusau**
Used the cs765_1_of_6.py as a basis along with sockets and multiple threads for SocketBraitenburg. Should run using the same SocketServer.

Also added a bash script to stop the motors

### CODE REVIEW

File: SocketClient.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def Main():
         if k == 'p':
             stop()
         if k == 'q':
-            exit()
+            break
```
Comment (dariusau) [inline SocketClient.py:212]: Doesn't actually close the socket if exit() is used

**Review by ministiy — COMMENTED** [review]
did you mean to comment this out? whoops ignore this one

File: SocketBraitenburg.py [hunk 2]
```diff
@@ -235,7 +237,7 @@ def startNewThread(name):
 print("Thread created")
 
 try:
-    output_file = writer.openFile()
+    'output_file = writer.openFile()'
```
Comment (ministiy) [inline SocketBraitenburg.py]: did you mean to comment this change?

File: SocketClientBraitenburg.py [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+def right(state):  # neater use of 'if' follows:
+    global SENSOR_GAIN
+    if state:
+        SENSOR_GAIN += 0.05
+        print('SENSOR_GAIN:', SENSOR_GAIN)
+
+# ===============================================
+
+def left(state):
+    global SENSOR_GAIN
```
Comment (ministiy) [inline SocketClientBraitenburg.py]: is this the same SENSOR_GAIN set up in ev3devsetup?

File: SocketClientBraitenburg.py [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+def up(state):
+    global OUTPUT_GAIN
+    if state:
+        OUTPUT_GAIN += 0.5
+        print('OUTPUT_GAIN:', OUTPUT_GAIN)
+
+# ===============================================
+
+def down(state):
+    global OUTPUT_GAIN
```
Comment (ministiy) [inline SocketClientBraitenburg.py]: same question as SENSOR_GAIN

File: SocketClientBraitenburg.py (hunk 3 above)
Comment (dariusau) [reply SocketClientBraitenburg.py]: Yup

File: SocketClientBraitenburg.py (hunk 4 above)
Comment (dariusau) [reply SocketClientBraitenburg.py]: Yup

File: SocketClientBraitenburg.py (hunk 3 above)
Comment (dariusau) [reply SocketClientBraitenburg.py]: I just fused the cs765 and the SocketClient together - thereby not being efficient

**Pull request merged**